pkg/frontend/openshiftcluster_putorpatch.go

```diff
@@ -133,6 +134,10 @@ func (f *frontend) _putOrPatchOpenShiftCluster(ctx context.Context, r *http.Requ
 	if isCreate {
 		err = staticValidator.Static(ext, nil)
 	} else {
+		timeoutCtx, cancel := context.WithTimeout(ctx, 10*time.Second)
+		defer cancel()
+		f.ocEnricher.Enrich(timeoutCtx, doc.OpenShiftCluster)
```

Comment:
does this need to be pulled forward for the PATCH case?  i.e. we need to merge onto an enriched document?  To around line 105?

Reply:
This PR brings forward the caching requirement.
With this PR, if I do
PUT(create)->GET->GET->DELETE, at deleting I'll see the schema as I created it
if I do
PUT(create)->GET->PUT->GET->DELETE, at deleting I'll see the schema as I last PUT it

Reply:
TL;DR in a follow-up PR we should consider doing a DB write after all the enrich tasks have completed